### PR TITLE
Restandardize Screen Dimension Notation

### PR DESCRIPTION
**Pull request by cpsauer**
Hi, @lmirosevic! Thanks again for such a wonderful library.

This PR tries to clean up an inconsistent enum naming problem I'd noticed while upgrading to the latest version.

Enum naming had been originally standardized around #[p#]Inch, but had drifted to be inconsistent, I assume from PRs. This PR fixes that before things diverge further, hopefully batching efforts with the backwards incompatible changes in the last release.

As examples, this restores consistency with GBDeviceDisplay4Inch and GBDeviceModeliPadPro9p7Inch.

Thanks for your consideration!
Chris

P.S. If we're trying to further batch incompatible changes, I think there might be an opportunity for improvement for, for example GBDeviceModeliPadPro12p9Inch31TB. Probably the 1TB has to be last for the version number, 3, to make sense, but it reads ambiguously like 31TB. I'd propose, perhaps GBDeviceModeliPadPro12p9Inch3_1TB. Whether or not you like that idea, please still just merge this PR, and one of us can follow on with another PR for it thereafter.

### CODE REVIEW

**Comment by cpsauer** [comment]
One other thought: Could be fun to add that hacktoberfest label to the repo. Just added it to the ones I maintain.

**Comment by cpsauer** [comment]
[Happy to help with the new iPads released today, but I figure we should handle this one first, since the naming will depend on it.]

**Comment by lmirosevic** [comment]
Thanks @cpsauer this is a great PR. Merging this. I will do a major version bump for this one.

**Comment by lmirosevic** [comment]
I've released this as 7.0.0 to CocoaPods

**Comment by cpsauer** [comment]
Sweet! Thanks, @lmirosevic, for keeping this running :)

To tie up the dangling threads and get to next steps:
- What are your thoughts on the P.S. about 1TB iPad models? 
  - Refined proposal: After some more thought, I think I'd actually propose that we merge the 1TB models in under the same enum as their smaller storage counterparts. For example, resolve the hard to read GBDeviceModeliPadPro12p9Inch31TB -> GBDeviceModeliPadPro12p9Inch3. 
  - Reasoning: I'd suggest the above because we're already merging wifi and cellular models, and because the 1TB models aren't otherwise broken out physically in Apple's [physical spec listings](https://developer.apple.com/accessories/Accessory-Design-Guidelines.pdf). That implies that their physical interface is the same--presumably the change is just an internal retiling, perhaps with an extra set of pads for another SSD IC. If I'm wrong, and there's an important use for this differentiation, please do say something!
  -  Otherwise, would you be open to a PR, removing the hard to parse (and probably not so useful) 1TB enum endings?
- Any chance you'd be down to add the octoberfest topic to the repo. That's not why I'm doing these, but it'd be fun to get the t-shirt this year, giving I'll be contributing anyway.

**Comment by lmirosevic** [comment]
Good call on all of those!

I've also added the hacktoberfest topic 👍

Thanks a lot for the PRs!

**Comment by cpsauer** [comment]
Thank you, @lmirosevic, for all you do!
